src/ColorClock.elm

```diff
@@ -66,20 +66,16 @@ view model =
         ]
 
 
-update : Msg -> Model -> ( Model, Cmd Msg )
+update : Msg -> Model -> Model
 update msg model =
     case msg of
         ChangeTime newTime ->
-            ( { model | time = newTime }
-            , Cmd.none
-            )
+            { model | time = newTime }
 
         ChangeTimeZone newZone ->
-            ( { model | zone = newZone }
-            , Cmd.none
-            )
+            { model | zone = newZone }
 
 
 subscriptions : Model -> Sub Msg
-subscriptions model =
+subscriptions always =
```

Comment:
That's not what I mean - you're just overshadowing `always` with your argument.
I meant:
```elm
subscriptions : Sub Msg
subscriptions =
   Time.every 1000 ChangeTime
```
and then in `main`:
```elm
Browser.element
    ...
    , subscriptions = always subscriptions
    ...
```
This way, you use `always` to consume the unused `model` `Browser.element`'s `subscriptions` receives.